### PR TITLE
multi: add autopilot stubs

### PR DESCRIPTION
**Pull request by ellemouton**
Necessary updates for compatibility with https://github.com/lightninglabs/lightning-terminal/pull/487

That PR needs to be merged & a new litd version released before this can be merged

### CODE REVIEW

**Review by kaloudis — APPROVED** [review]

**Review by jamaljsr — APPROVED** [review]
tACK 🚀 Let's go!

**Comment by ellemouton** [comment]
Thanks for the review guys! I will merge this after new `lightning-terminal` and `lightning-terminal/autopilotserverrpc` tags have been pushed.

File: cmd/wasm-client/go.mod [hunk 1]
```diff
@@ -160,4 +160,6 @@ replace github.com/lightninglabs/lightning-node-connect => ../../
 
 replace github.com/lightninglabs/lightning-node-connect/hashmailrpc => ../../hashmailrpc
 
+replace github.com/lightninglabs/lightning-terminal/autopilotserverrpc => github.com/lightninglabs/lightning-terminal/autopilotserverrpc v0.0.0-20230215070548-46a1d7e31713
```
Comment (guggero) [inline cmd/wasm-client/go.mod:163]: Can be replaced by the correct tag now.

**Review by guggero — APPROVED** [review]
LGTM :tada:

**Pull request merged**